music_assistant/server/controllers/media/albums.py

```diff
@@ -292,7 +292,6 @@ async def _get_provider_album_tracks(
         items = []
         for track in await prov.get_album_tracks(item_id):
             assert isinstance(track, AlbumTrack)
-            assert track.track_number
```

Comment:
this is not allowed

Reply:
btw: this assert is here for the reason that we often make this mistake

Reply:
This was removed per our discussion on Discord because of 0 numbered pre tracks. Before I removed this I was setting the 0 tracks to 99 but that makes them out of order and there could be the case that there is a pre-track (0) and over 98 tracks.

https://discord.com/channels/753947050995089438/1083871908317102150/1198446101376802817